docs/source/backgroundechemtechniques.rst

```diff
@@ -0,0 +1,75 @@
+Electrochemical Techniques
+===========================
+
+This page is designed to help users choose the best electrochemical testing technique for a given experiment. The JUAMI
+potentiostat offers three techniques for testing electrochemical systems - linear sweep voltammetry, cyclic
+voltammetry, and chronoamperometry. An introduction to each technique is below.
+
+Linear Sweep Voltammetry (LSV)
+------------------------------
+
+Linear sweep voltammetry applies a potential that sweeps from one potential to a final potential and measures the
+current response from the electrochemical system. This technique allows a user to specifically analyze an oxidation
+(anodic) or a reduction (cathodic) reaction based on the direction (+/-) of the potential scan.
+
+User Inputs:
+
+* start_voltage (V)
+* end_voltage (V)
+* sweep_rate (mV/s)
+
+Applied Waveform:
+
+.. image::
+
+Measured Waveform:
+
+.. image::
+
+After completion of an LSV experiment the data file contains time, voltage and current. Using this information, one
+can plot current vs. voltage, shown below, which is typically the most useful way to plot the results from a LSV scan. Additional
+LSV measurements with different sweep rates can be used to investigate changes in electrochemical behavior as a function
+rate.
+
+.. image:: lsv_ferricyanide_001.jpg
```

Comment:
This image isn't showing up for me, not sure why. Can see it in image folder though.